src/main/java/com/eternitywall/ots/Timestamp.java

```diff
@@ -233,13 +233,15 @@ public TimeAttestation shrink() throws Exception {
         }
 
         // Remove attestation if not min attestation
+        boolean shrinked = false;
         for (Iterator<Entry<Op, Timestamp>> it = this.ops.entrySet().iterator(); it.hasNext();) {
```

Comment:
isn't `for (Map.Entry<Op, Timestamp> entry : timestamps.entrySet()) ` the same?
If yes, I think it's better